recipe/meta.yaml

```
@@ -9,32 +8,33 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: faa507115731742e4eb0fbfec7be41310efcc46523afc188440b2d26ed48d361
 
 build:
   number: 0
+  # Windows skip is due to snowflake only build of tiledb 2.26.2
+  skip: True  # [py<39 or s390x or win]
+  ignore_run_exports:
+    - numpy
+
 requirements:
   build:
-    - {{ compiler('cxx') }}
-    - {{ stdlib("c") }}
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython >=3.0                            # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - pybind11                               # [build_platform != target_platform]
-    - cmake
+    - pybind11
+    - cmake >=3.15
     - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
   host:
     - pip
     - wheel
     - setuptools
-    - setuptools_scm
+    - setuptools-scm
     - scikit-build-core
     - python
-    - cython >=3.0
-    - numpy
+    - cython >=3
+    - numpy >=2.0.1
     - pybind11
-    - tiledb >=2.26.0,<2.27
+    - tiledb >=2.26
   run:
     - python
     - packaging
```

```suggestion
    - packaging
    - numpy >=1.25
```
see https://github.com/TileDB-Inc/TileDB-Py/blob/0.32.5/pyproject.toml#L40